packages/public/server/src/module/Api/src/Listener/LinkServiceListener.php

```diff
@@ -52,6 +52,13 @@ public function onChange(Event $e)
         $this->getApiManager()->setUuid($term);
     }
 
+    public function onSortChildren(Event $e)
+    {
+        /** @var UuidInterface $parent */
+        $parent = $e->getParam('parent');
+        $this->getApiManager()->removeUuid($parent->getId());
```

Comment:
`setUuid()` does not work here since `$parent` does not hold the new order of children yet. I guess the changes are not flushed and I haven't found a quick way to do this. @inyono If you want to have `setUuid()` at this place, let's open a follow up issue. However I want to discuss in the dev meeting whether using `removeCache` is the smarter solution...

Reply:
> However I want to discuss in the dev meeting whether using `removeCache` is the smarter solution...

Yeah, let's discuss that. This change would at least go against one of the main principles of the listeners. Besides making sure that the cache is up-to-date (vs. a time-based cache for example), it should also improve request times (which would not be the case if we just remove the cache).